### PR TITLE
add semantic versioning info

### PR DESCRIPTION
**Pull request by pravi**
for #43

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,6 +98,16 @@ end
 ```
 
 
+## Semantic Versioning
+
+This project conforms to [semver](http://semver.org/). As a result of this policy, you can (and should) specify a dependency on this gem using the [Pessimistic Version Constraint](http://guides.rubygems.org/patterns/) with two digits of precision. For example:
```
Comment (steveklabnik) [inline README.md]: can you wrap this to 80 chars?

File: README.md [hunk 2]
```diff
@@ -98,6 +98,16 @@ end
...
+
+spec.add_dependency 'request_store', '~> 1.0'
```
Comment (steveklabnik) [inline README.md]: can you put this in code blocks?

File: README.md [hunk 3]
```diff
@@ -98,6 +98,16 @@ end
...
+
+This means your project is compatible with request_store 1.0 up until 2.0. You can also set a higher minimum version:
```
Comment (steveklabnik) [inline README.md]: same, wrapping

File: README.md [hunk 4]
```diff
@@ -98,6 +98,16 @@ end
 ```
```
Comment (steveklabnik) [inline README.md]: mind making this one newline rather than two? I know it was there already, but that's weird

File: README.md [hunk 5]
```diff
@@ -98,6 +98,16 @@ end
...
 
+## Semantic Versioning
+
+This project conforms to [semver](http://semver.org/). As a result of this policy, you can (and should) specify a dependency on this gem using the [Pessimistic Version Constraint](http://guides.rubygems.org/patterns/) with two digits of precision. For example:
+
+spec.add_dependency 'request_store', '~> 1.0'
+
+This means your project is compatible with request_store 1.0 up until 2.0. You can also set a higher minimum version:
+
+spec.add_dependency 'request_store', '~> 1.1'
```
Comment (steveklabnik) [inline README.md]: and code block here

**Comment by steveklabnik** [comment]
hehe, sorry for too many nits :( after they're done, i'll merge

**Comment by pravi** [comment]
suggested formatting changes applied

**Comment by steveklabnik** [comment]
👍❤️

**Pull request merged**